model/src/main/kotlin/config/FileArchiverConfiguration.kt

```diff
@@ -33,14 +37,42 @@ data class FileArchiverConfiguration(
     /**
      * Configuration of the [FileStorage] used for archiving the files.
      */
-    val storage: FileStorageConfiguration
-)
+    val fileStorage: FileStorageConfiguration? = null,
+
+    /**
+     * Configuration of the [PostgresFileArchiverStorage] used for archiving the files.
+     */
+    val postgresStorage: PostgresStorageConfiguration? = null
+) {
+    init {
+        if (fileStorage != null && postgresStorage != null) {
+            log.warn {
+                "'fileStorage' and 'postgresStorage' are both configured but only one storage can be used. Using " +
+                        "'fileStorage'."
+            }
+        }
```

Comment:
But shouldn't we then make this a `require()`-ment like we do in other places, instead of implementing a fallback?